core/src/DAL/index.js

```
@@ -0,0 +1,134 @@
+        }
+        if (this.#lazyTmDB) {
+            this.#lazyTmDB.close();
+        }
+    }
+}
+
+export function createSQLObjectTransformer(jsonProps, spreadingProps = []) {
+    return {
+        encode(obj) {
+            for (const key of jsonProps) {
+                if (obj.hasOwnProperty(key) && typeof obj[key] === 'object' && obj[key] !== null) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)